en/includes/guides/organization-management/email-domain-based-organization-discovery.md

```diff
@@ -1,193 +1,212 @@
+=== "Using the API"
 
-### Using the API
+    !!! note "Before you begin"
 
-1. [Get an access token]({{base_path}}/apis/#oauth-based-authentication) with the `internal_organization_config_add` scope.
+        Ensure you have configured email address as a username option. Learn more about [multi-attribute login]({{base_path}}/guides/authentication/multi-attribute-login/).
 
-2. Use the obtained access token to execute the following cURL.
+    To do so,
 
-    ``` curl
-    curl --location --request POST 'https://{{ host_name }}{{ organization_path_param }}/api/server/v1/organization-configs/discovery' \
-    --header 'Content-Type: application/json' \
-    --header 'Authorization: Bearer <access_token>' \
-    --data '{
-        "properties": [
-            {
-                "key": "emailDomain.enable",
-                "value": true
-            }
-        ]
-    }'
-    ```
+    1. [Get an access token]({{base_path}}/apis/#oauth-based-authentication) with the `internal_organization_config_add` scope.
+
+    2. Use the obtained access token to execute the following command.
+
+        ``` curl
+        curl --location --request POST 'https://{{ host_name }}{{ organization_path_param }}/api/server/v1/organization-configs/discovery' \
+        --header 'Content-Type: application/json' \
+        --header 'Authorization: Bearer <access_token>' \
+        --data '{
+            "properties": [
+                {
+                    "key": "emailDomain.enable",
+                    "value": true
+                }
+            ]
+        }'
+        ```
 
-## Configure email domains for organizations
+    5. [Map email domains to organizations](#map-email-domains-to-organizations).
 
-### Prerequisites
 
-You need to:
-    - enable email domain based organization discovery for the organization (root).
-    - have registered organizations in {{ product_name }}. See [Manage organizations]({{base_path}}/guides/organization-management/manage-organizations/) for instructions.
+## Map email domains to organizations
 
-### Using the Console
+Mapping an email domain to an organization allows for a customized user experience, but it is not mandatory. When an organization does not have a mapped email domain, the behavior changes as follows:
 
-1. On the {{ product_name }} Console, go to **Login & Registration**.
-2. Click **Email Domain Discovery** under **Organization Settings**.
-3. Click **+Assign Email Domains**.
+- If an organization registers an email domain mapping,
+    - a user can only onboard to the organization if the user's email domain matches one of the domains claimed by the organization.
+    - Just-In-Time (JIT) provisioning during federated authentication only occurs if the user's email domain matches one of the domains claimed by the organization.
```

Comment:
Let's mention that federation authentication with unmatched email domains will be restricted as well